src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -84,6 +84,7 @@
     <Compile Include="System\ExecutionEngineExceptionTests.cs" />
     <Compile Include="System\ExternalExceptionTests.cs" />
     <Compile Include="System\FieldAccessExceptionTests.cs" />
+    <Compile Include="System\FormattableStringTests.cs" />
```

Comment:
This is using new APIs that are only present now in .NET Core, but these tests are executed on .NET Framework as well.  You'll need to conditionalize including this file when building for netcoreapp  There should be a section for that elsewhere in the .csproj.